notebooks/crop-mask

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
What's this notebook doing here?

Reply:
Thanks for the observation; this is a mistake.